src/main/java/forceitembattle/manager/ItemDifficultiesManager.java

```diff
@@ -14,6 +15,8 @@ public class ItemDifficultiesManager {
     List<Material> medium;
     List<Material> hard;
 
+    List<DescriptionItem> descriptionItems;

```

Comment:
why no HashMap<Material, DescriptionItem>? Would be more efficient or not?

Reply:
Don't know exactly why, that would be redundant for me, since the record of DescriptionItem stores the material anyway or do I see it wrong?

Reply:
But now you have a running time of O(n) when getting the description. You iterate over all the items in the List and search for the correct one. If you use a HashMap instead it's running time O(1) so instant (the server only has to calculate the hash of the Material)

Reply:
Oh yea you're right. Thanks didnt think about that. Should be hopefully fixed now

Reply:
oh wait no I made a mistake

Reply:
Alright, now it should be fixed, lmk if there are issues with my commit